pkg/models/reading.go

```diff
@@ -28,20 +26,20 @@ import (
  * Struct for the Reading object in EdgeX
  */
 type Reading struct {
-	Id       bson.ObjectId `bson:"_id,omitempty"`
-	Pushed   int64         `bson:"pushed" json:"pushed"`   // When the data was pushed out of EdgeX (0 - not pushed yet)
-	Created  int64         `bson:"created" json:"created"` // When the reading was created
-	Origin   int64         `bson:"origin" json:"origin"`
-	Modified int64         `bson:"modified" json:"modified"`
-	Device   string        `bson:"device" json:"device"`
-	Name     string        `bson:"name" json:"name"`
-	Value    string        `bson:"value" json:"value"` // Device sensor data value
+	Id       string        `json:"id"`
+	Pushed   int64         `json:"pushed"`   // When the data was pushed out of EdgeX (0 - not pushed yet)
+	Created  int64         `json:"created"` // When the reading was created
+	Origin   int64         `json:"origin"`
+	Modified int64         `json:"modified"`
+	Device   string        `json:"device"`
+	Name     string        `json:"name"`
+	Value    string        `json:"value"` // Device sensor data value
 }
 
 // Custom marshaling to make empty strings null
 func (r Reading) MarshalJSON() ([]byte, error) {
 	test := struct {
-		Id       bson.ObjectId `json:"id"`
+		Id       *string       `json:"id"`
```

Comment:
Same as Event, can you add `omitempty` tag?

Reply:
Will be included in updated PR